invoicer_project/static/js/registration_request.js

```diff
@@ -0,0 +1,302 @@
+const passwordMinLength = 8;
+const passwordMaxLength = 15;
+const nameSurnMaxLength = 35;
+const host = "http://127.0.0.1:8000";
+let csrfToken = document.getElementsByName('csrfmiddlewaretoken')[0].value;
+
+const emailForm = document.getElementById("email_input_registration_page");
+const nameForm = document.getElementById("name_input_registration_page");
+const surnameForm = document.getElementById("surname_input_registration_page");
+const passForm = document.getElementById("password_input_registration_page");
+const repeatPassForm = document.getElementById("repeat_password_input_registration_page");
+
+
+function validateRegistration() {
+    removeMaxHeightAttribute();
+    let names = validateNameAndSurname();
+    let email = validateEmail();
+    let passwords = validatePasswords();
+    
+    checkAndSetNormalHeightNameAndSurnameInput();
+    checkAndSetNormalHeightPasswordAndRepeatPasswordInput();
+    return (email && passwords && names)
+}
+
+nameForm.addEventListener('input', () => {
+    nameForm.removeAttribute("errorText");
```

Comment:
Should implement checking if this attribute is present before removing it
(hasAttribute() ....)

Reply:
Why would you need that? That will simply invoke one more unnecessary if-caluse. The call of element.removeattribute removes specified attribute in case it exists and is safe since you dont't get an error in case attribute value is not specified.
Example: 
![image](https://user-images.githubusercontent.com/99320958/218246388-4af1310d-e268-4726-99fd-a271d525fbd1.png)
After clicking the button you'll get no errors but the attribute type is gonna be removed (while name attr doesn't exist)